### ISSUE TITLE
Meeting Mar 28th 

### ISSUE THREAD

**Issue by cleebp** [cleebp/csc-510-group-g#62]
Meeting monday march 28th, room 3314

**Comment by arjunmadan** [comment]
What time was this again? :stuck_out_tongue:

**Comment by reachspirit** [comment]
from 12pm to 2pm

**Comment by cleebp** [comment]
Hey guys, I posted this in the other issue but I have office hours till 1 so I'll be a bit late. That being said no one really shows up to these so I'll be in the issues trying to contribute where I can.

Thoughts on stuff to get done today:
- Store the data we did end up collecting into the google sheet "experimental data". Right now that sheet is nearly empty (I just filled in some ideas on the organization, its hard to come up with a way to organize the data by participant and by task) so we will need to fill it out with relevant things to care about and fill it in with the data we collected.
- Document all of the issues we ran into when testing! This is important, we can use those issues as data points themselves in our paper, all data is data :D Not sure where the best place to do this is, maybe in an issue and then also in a google doc or something? We need something we can look at while writing the paper so the temporary existence of issues is an issue... haha... puns...
- ... more stuff?
